.dockerignore

```diff
@@ -0,0 +1,29 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+lerna-debug.log*
+
+**/node_modules
+# **/dist
```

Comment:
I think it's better to have this ignored

Reply:
yep you right